### DISCUSSION TITLE
assist

### DISCUSSION THREAD

**Discussion by ashrafmahmoude** [bbc/bmx#56] [General]
Are there any documents for using the library?
I am trying to download a program to play an encrypted video, and unfortunately I am new to the world of C++

**Comment by philipnbbc** [comment]
There is no documentation other than what's provided here. I don't think bmx will help you with encrypted videos as it doesn't have support for decryption. If you need it for something else then the binary executables in the [releases](https://github.com/bbc/bmx/releases) may be of use. Otherwise try FFmpeg as that can do a lot more.
